module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInfoTest.php

```diff
@@ -179,6 +179,38 @@ public function testFindContainingThemeWithMixin()
         $this->assertEquals('child', $ti->findContainingTheme('child.txt'));
     }
 
+    /**
+     * Test findInThemes()
+     *
+     * @return void
+     */
+    public function testFindInThemes()
+    {
+        $ti = $this->getThemeInfo();
+        $ti->setTheme('child');
+        $files = $ti->findInThemes(
+            [
+                'templates/content/*.phtml',
+                'templates/content/*.md'
+            ]
+        );
+        $this->assertEquals('parent', $files[0]['theme']);
```

Comment:
Feel free to disagree if you took this path intentionally, but I'd be inclined to be a little more specific about the assertions, either by doing one big assertEquals on the whole $files variable, or else by adding `$this->assertEquals(3, count($files))`, just to be sure we get what we expect, and ONLY what we expect. I'm happy to make the change if you agree with the principle. :-)

Reply:
Adding the count check makes sense. I decided to chicken out from one big assert since one of the returned fields is an absolute path. I suppose I can dig it up though, if it would be better that way.

Reply:
I'm fine with just adding the count for now, rather than spending a lot of time recreating that path. :-)

Reply:
Done. :)